### PR TITLE
Issue#29

### PR DESCRIPTION
**Pull request by alexstandiford**
- Added a database flag that removes the sidebar form if a user signs up for the email list.
- Fixed a typo
- Corrected the form's markup to better-match W3 standards

### CODE REVIEW

**Comment by davekiss** [comment]
@alexstandiford What size screen are you viewing this on? The display starts getting wonky for sub-1000px viewports.

File: lib/admin/base.php [hunk 1]
```diff
@@ -60,6 +60,32 @@ public function icons_url() {
...
   }
+  
+  
+  /**
+   * Checks if the user has already signed up for the email list
+   * 
+   * @return boolean
+   */
+  public static function has_user_signed_up() {
+    return get_option('vimeography_has_user_signed_up');
```
Comment (davekiss) [inline lib/admin/base.php]: Please use `get_site_option()` here for multisite purposes

File: lib/ajax.php [hunk 2]
```diff
@@ -24,4 +25,10 @@ public function vimeography_ajax_get_cached_videos() {
 
     die;
   }
+  
+  //Flags the user if they have signed up to the email list
+  public function vimeography_toggle_signup_form() {
+    update_option('vimeography_has_user_signed_up',true);
```
Comment (davekiss) [inline lib/ajax.php]: Please use `update_site_option()` here for multisite purposes

**Comment by alexstandiford** [comment]
@davekiss - we could use flexbox, sure.

**Comment by alexstandiford** [comment]
Check this out and let me know what you think. I don't think it's _great_ on mobile, but I definitely think it is at least _as good_ as it was before I added the CTA.

**Pull request closed without merging**